README.md

```diff
@@ -8,25 +8,16 @@ if (moreUsers) {
 }
 ```
 
-### Server Side Usage
 
-#### NConf-Based Config
+#### Argument-Based Config
 
-To get started with neff you'll want to make sure you have setup nconf to contain a `features` setting, which is an object with feature names and either `true` or `false` values. I'm currently doing this using a JSON file and my config looks like this:
+If you'd instead like to pass the features as parameters you can call the factory method to merge the arguments passed to the method with any (optional) nconf configuration you utilize in your project.
 
-```json
-"features": {
+```
+app.use(neff.factory({
```

Comment:
So not sure how much it matters, but because `neff` is a singleton, each time you include the middleware you wipe the entire list. Maybe it would be better if each `.factor()` returned its own version of `neff`? We don't need `new`, but maybe it could contain it's features in its own scope? 

Let's talk about if this is needed.